test/ruby/scenarios.json

```diff
@@ -1,17 +1,36 @@
 {
-    "install_additional_ruby": {
-        "image": "ubuntu:focal",
+    "non_root_user": {
+        "image": "mcr.microsoft.com/devcontainers/base:bullseye",
```

Comment:
I noticed that the Ruby image used a Ruby Docker image as its base. That seemed strange to me. Not sure if it impacts the test scenarios, though. The Node image also uses a Node Docker image as its base, but the Node Feature tests test on Microsoft base devcontainer images. Perhaps a broader concern?